configure.ac

```diff
@@ -5186,6 +5186,11 @@ if test "x$with_libupsclient" = "xyes"; then
     [with_libupsclient="no (symbol upscli_connect not found)"]
   )
 
+  AC_CHECK_LIB([upsclient], [upscli_init],
+    [AC_DEFINE(WITH_UPSCLIENT_27, 1, [At least version 2-7])],
```

Comment:
I looked at a number of other AC_DEFINE calls within the same file and none of them have their arguments quoted. Most had square brackets around them though. Is this what you meant?

I'll add square brackets but not quotes to keep with the style of the rest of the file.